server/views/pages/dashboard.njk

```diff
@@ -134,7 +140,7 @@
         }) }}
 
         {% if not noFiltersSupplied %}
-          <a href="?" class="govuk-link--no-visited-state" id="clearFilter">Clear</a>
+          <a href="?" class="govuk-link--no-visited-state" id="clearFilter">Clear filters</a>
```

Comment:
turns out this needs the `govuk-link` class as well (before the one already there)